src/Libraries/DSCPython/Encoders/ListEncodeDecoder.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace DSCPython.Encoders
 {
-    internal class ListEncoder : IPyObjectEncoder, IPyObjectDecoder
+    internal class ListEncoderDecoder : IPyObjectEncoder, IPyObjectDecoder
```

Comment:
I see obsolete messages on `IPyObjectEncoder` and Decoder saying they are unstable and can be changed in the next minor release - something to keep note of unless there is an alternative interface already existing that we can use and replace right away.

Reply:
The custom encoder/decoder API appeared with 2.5, which is the latest minor release of Python.NET. My guess is they might make changes to it in version 3.0 so you bring a good point.

Before this API there was no way to do something like this AFAIK. We chose to use the API where possible, even if it is unstable, because otherwise we would have to modify Python.NET even more. Also, from these encoders/decoders you could target custom types, like DS Dictionary, which is another plus.